packages/api/src/typeDefs/propertyValue.graphql

```diff
@@ -0,0 +1,4 @@
+type StorePropertyValue {
```

Comment:
Does `StoreAdditionalProperty` make more sense for you? StorePropertyValue has a value and a name? Seems weird.